QuickDeployment/QuickDeploymentSetup.md

```diff
@@ -0,0 +1,135 @@
+# Quick Deployment Environment Setup
+
+This document contains instructions for importing the QuickDeploy appliance/VM, or creating a VM and attaching the QuickDeploy disk image to it.
+You will receive a download link via email for an archive of the VM image. Once you have this downloaded, it will be ready for extraction and import into your environment.
+
+## Setup Considerations
+
+The following are points to keep in mind during initial setup:
+
+* You will need access to AWS to download the environment (specifically `s3.amazonaws.com`).
+* Hostname/FQDN changes will invalidate all scripts and certificates. Thus, if you plan to change the hostname, you must do so prior to running any setup scripts. If you run into issues, refer to the README file on the desktop of the VM.
+* Self-signed certificates are generated by default. If you plan to use your own certificates, please reach out to Support for assistance.
+* The back-end database is configured with no outbound connections - if you plan to change this, please reach out to Support for assistance.
+* If you intend to use Nexus outside of your corporate network without the use of a VPN, you will be required to configure RBAC on the repositories housed inside of the repository server. This is to ensure that the packages stored on the server are not publicly accessible without authentication.
```

Comment:
Is there information from Nexus that we can supply here to help people assess the risks and configure Nexus for being publicly accessible?